WakeLock/WakeLock_landing_landing.html

```diff
@@ -6,15 +6,15 @@
+<p>Most devices by default turn off their screen after a specified amount of time to prolong the life of the hardware. Modern devices do this to save on battery power. Whilst this is a useful feature, some applications need the screen to stay awake to be their most useful.</p>
 
 <p>The Screen Wake Lock API prevents the screen from turning off, dimming or locking. It allows for a simple platform based solution which up until now could only be achieved via workarounds which were potentially power hungry. Only visible (active) documents can acquire the screen wake lock.</p>
 
-<p>There are plenty of use cases for keeping a screen on, including reading an ebook, following a recipe, presenting to an audience, scanning a qr/barcode or applications that use voice or gesture control, rather than tactile input (the default way to keep a screen awake).</p>
+<p>There are plenty of use cases for keeping a screen on, including reading an ebook, following a recipe, presenting to an audience, scanning a QR/barcode or applications that use voice or gesture control, rather than tactile input (the default way to keep a screen awake).</p>
 
-<p>You acquire a {{domxref('WakeLockSentinel')}} object by calling the {{domxref('WakeLock.request','navigator.wakeLock.request()')}} {{domxref('Promise')}} based method that resolves if the platform allows it. A request may be rejected for a number of reasons, including system settings (such as power save mode or low battery level) or the document not being active or visible.</p>
+<p>You acquire a {{domxref('WakeLockSentinel')}} object by calling the {{domxref('WakeLock.request','navigator.wakeLock.request()')}} {{jsxref('Promise')}} based method that resolves if the platform allows it. A request may be rejected for a number of reasons, including system settings (such as power save mode or low battery level) or if the document is not active or visible.</p>
 
-<p>The sentinel is attached to the underlying system wake lock and can be released either by the system, again if the battery power is too low or the document becomes not visible or inactive. It can also be released manually via the release() method. It is good practice to store a reference to the sentinel object to control release later and also to reacquire the system wake lock if need be.</p>
+<p>The sentinel is attached to the underlying system wake lock. It can be released by the system, again if the battery power is too low or the document is not active or visible. It can also be released manually via the {{{{domxref('WakeLockSentinel.release()')}}}} method. It is good practice to store a reference to the sentinel object to control release later and also to reacquire the lock if need be.</p>
```

Comment:
```suggestion
<p>The sentinel is attached to the underlying system wake lock. It can be released by the system, again if the battery power is too low or the document is not active or visible. It can also be released manually via the {{domxref('WakeLockSentinel.release()')}} method. It is good practice to store a reference to the sentinel object to control release later and also to reacquire the lock if need be.</p>
```